src/Tribe/Repositories/Attendee/Commerce.php

```diff
@@ -77,4 +115,149 @@ public function checked_in_keys() {
 		return $this->limit_list( $this->key_name, parent::checked_in_keys() );
 	}
 
+	/**
+	 * Set up the arguments to set for the attendee for this provider.
+	 *
+	 * @since TBD
+	 *
+	 * @param array                         $args          List of arguments to set for the attendee.
+	 * @param array                         $attendee_data List of additional attendee data.
+	 * @param Tribe__Tickets__Ticket_Object $ticket        The ticket object or null if not relying on it.
+	 *
+	 * @return array List of arguments to set for the attendee.
+	 */
+	public function setup_attendee_args( $args, $attendee_data, $ticket = null ) {
+		// Set default attendee status.
+		if ( ! isset( $args['attendee_status'] ) ) {
+			$args['attendee_status'] = 'completed';
```

Comment:
Should we use `Tribe__Tickets__Commerce__PayPal__Stati::$completed`?

Reply:
Updated